docs/ios_xcuitest.md

```diff
@@ -86,5 +86,18 @@ xpaths("//some xpaths")
 - Specs by test code
     - https://github.com/appium/appium-xcuitest-driver/blob/master/test/functional/basic/gesture-e2e-specs.js
 
+### Workaround
+- `mobile:` commands depends on WDA and Apple's framework and the behaviour depends on them.
+- Sometimes issues occur such as "doubleTap isn't tapping #548"
+    - workaround for it:
+        ```ruby
+        def double_tap(element)
+          # Find the object, then go to the middle of it, and double-tap
+          tap_x = element.location.x + element.size.width * 0.50
+          tap_y = element.location.y + element.size.height * 0.50
+          execute_script 'mobile: doubleTap', x: tap_x, y: tap_y
```

Comment:
Thanks!

I checked selenium-webdriver for Ruby about **getRect**.
We can call it like the following command if we use selenoum-webdriver 3.4.0+, under 3.4.0 can't.

```
[2] pry(#<ios/element/text>)> t = text('siti')
=> #<Selenium::WebDriver::Element:0x40d05fd0c9668d4c id="51D1ADA5-D079-448B-BABE-1C6ABEE94A32">
[3] pry(#<ios/element/text>)> t.rect
=> #<struct Selenium::WebDriver::Rectangle x=17, y=647, width=105, height=27>
```

So, I'll update the workaround in two cases, one is using **rect** and another isn't.